translations/locales/fr-CA/translations.json

```diff
@@ -0,0 +1,591 @@
+      "PlayOnlyVisualSketchARIA": "Exécuter seulement le croquis visuel",
+      "StopSketchARIA": "Arrêter le croquis",
+      "EditSketchARIA": "Éditer le nom du croquis",
+      "NewSketchNameARIA": "Nouveau nom de croquis",
+      "By": " par "
+    },
+    "Console": {
+      "Title": "Console",
+      "Clear": "Effacer",
+      "ClearARIA": "Effacer la console",
+      "Close": "Fermer",
+      "CloseARIA": "Fermer la console",
+      "Open": "Ouvrir",
+      "OpenARIA": "Ouvrir la console"
+    },
+    "Preferences": {
+      "Settings": "Paramètres",
+      "GeneralSettings": "Paramètres généraux",
+      "Accessibility": "Accessibilité",
+      "Theme": "Thème",
+      "LightTheme": "Clair",
+      "LightThemeARIA": "Thème clair activé",
+      "DarkTheme": "Sombre",
+      "DarkThemeARIA": "Thème sombre activé",
+      "HighContrastTheme": "Contraste élevé",
+      "HighContrastThemeARIA": "Thème contraste élevé activé",
+      "TextSize": "Taille du texte",
+      "DecreaseFont": "Diminuer",
+      "DecreaseFontARIA": "diminuer la taille de la police",
+      "IncreaseFont": "Augmenter",
+      "IncreaseFontARIA": "augmenter la taille de la police",
+      "Autosave": "Sauvegarde automatique",
+      "On": "Activer",
+      "AutosaveOnARIA": "sauvegarde automatique activée",
+      "Off": "Désactiver",
+      "AutosaveOffARIA": "sauvegarde automatique désactivée",
+      "AutocloseBracketsQuotes": "Fermeture automatique des crochets et des guillemets",
+      "AutocloseBracketsQuotesOnARIA": "fermeture automatique des crochets et des guillemets activée",
+      "AutocloseBracketsQuotesOffARIA": "fermeture automatique des crochets et des guillemets désactivée",
+      "WordWrap": "Retour à la ligne automatique",
+      "LineWrapOnARIA": "retour à la ligne automatique activé",
+      "LineWrapOffARIA": "retour à la ligne automatique désactivé",
+      "LineNumbers": "Numéros de lignes",
+      "LineNumbersOnARIA": "numéros de lignes activés",
+      "LineNumbersOffARIA": "numéros de lignes désactivés",
+      "LintWarningSound": "Son d'alarme Lint",
+      "LintWarningOnARIA": "son d'alarme Lint activé",
+      "LintWarningOffARIA": "son d'alarme Lint désactivé",
+      "PreviewSound": "Tester le son",
+      "PreviewSoundARIA": "Tester le son",
+      "AccessibleTextBasedCanvas": "Canvas textuel accessible",
+      "UsedScreenReader": "Utilisé avec un lecteur de text",
+      "PlainText": "Texte brut",
+      "TextOutputARIA": "sortie texte activée",
+      "TableText": "Tableau de texte",
+      "TableOutputARIA": "sortie tableau de texte activée",
+      "Sound": "Son",
+      "SoundOutputARIA": "sortie son activée"
+    },
+    "KeyboardShortcuts": {
+      "Title": " Raccourcis clavier",
+      "ShortcutsFollow": "Suivi des raccourcis clavier d'édition",
+      "SublimeText": "Raccourcis clavier Sublime",
+      "CodeEditing": {
+        "Tidy": "Nettoyer",
+        "FindText": "Rechercher",
```

Comment:
Lastly, "Replace" had been added to the Keyboard Shortcuts ([see English](https://github.com/processing/p5.js-web-editor/blob/develop/translations/locales/en-US/translations.json#L182)):
```json
  "ReplaceTextMatch": "Replace Text Match",
```